python-frontend/src/main/resources/org/sonar/python/types/custom/cryptography/hazmat/primitives/asymmetric/rsa.pyi

```diff
@@ -0,0 +1,8 @@
+class RSAPublicKey:
```

Comment:
I think we're missing the "from typing import Any"

Reply:
Good point!